templates/terraform/examples/memcache_instance_basic.tf.erb

```diff
@@ -22,7 +22,6 @@ resource "google_service_networking_connection" "private_service_connection" {
 resource "google_memcache_instance" "<%= ctx[:primary_resource_id] %>" {
   provider = google-beta
   name = "<%= ctx[:vars]["instance_name"] %>"
-  region = "us-central1"
```

Comment:
Based on Riley's comments on the other PR, I would lean towards keeping `us-central1` here and for vpc access connector, unless there's a particular reason for removing them?

Reply:
Removing that infers the region from the provider. All tests run us-central1 where as the zone defaults to us-central1-a and that needed to be changed to be b

Reply:
👍 so they do. Thanks!